src/popup.js

```diff
@@ -29,7 +36,21 @@ const openModal = async (id) => {
   </div>
       <h3 class="summary">Synopsis</h3>
       <p class="series-desc">${shows[id].summary.replaceAll(/<b>|<\/b>|<p>|<\/p>/g, '')}</p>
+      <div class="comment-section">
+      <h4 class="center-comment">Comments (2)</h4>
```

Comment:
This line, where it says "Comments (2)" should display number of comments of this id (maybe the length of the array of the comments)

Reply:
okay, thank you very much for your review @mullenlucas , however this feature isn't part of the requirement for this task but i will make the change and commit it so you can approve